webapp/app/templates/lotse/form_person_b.html

```diff
@@ -24,15 +24,15 @@ <h3 class="mt-5 my-3">{{ _('form.lotse.person-header-addresse') }}</h3>
   <div id="form_address_fields">
     <div class="form-row-center">
       {{ macros.render_field(form.person_b_street) }}
-      {{ macros.render_field(form.person_b_street_number, 2) }}
-      {{ macros.render_field(form.person_b_street_number_ext, 4) }}
+      {{ macros.render_field(form.person_b_street_number, 2, field_div_classes="px-2") }}
     </div>
     <div class="form-row-center">
-      {{ macros.render_field(form.person_b_address_ext) }}
+      {{ macros.render_field(form.person_b_street_number_ext) }}
+      {{ macros.render_field(form.person_b_address_ext, field_div_classes="px-2") }}
```

Comment:
I am not sure if we need the padding on both sides or if `pl-2` would be enough? Same for all the other places with only two fields in one line